NuGet.config

```diff
@@ -3,6 +3,7 @@
   <packageSources>
     <clear />
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
+    <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
```

Comment:
What do we need this source for?

Reply:
Internal.AspNetCore.Sdk. We didn't need it with package lineup because we required 'build /t:Pin', which added a feed with this package. By removing this, we now need to go back to using the tools myget feed so a clean solution opens in VS w/o restore errors.